include/swift/SIL/SILFunction.h

```diff
@@ -288,6 +288,15 @@ class SILFunction
   /// that it may have unboxed capture (i.e. @inout_aliasable parameters).
   bool IsWithoutActuallyEscapingThunk = false;
 
+  // SWIFT_ENABLE_TENSORFLOW
+  /// Whether TFDeabstraction deabstracted this function.
+  ///
+  /// This is a temporary flag that IRGen uses to determine if the graph_ops in
+  /// this function are safe to lower. Once IRGen is powerful enough to lower
+  /// every graph_op that it receives, we should remove this flag.
+  /// TODO: Remove this flag.
+  bool TFDeabstracted = false;
```

Comment:
this patch is fine. for future reference, since it's a temp flag, it'd also be fine to make it a public member for simplicity.